src/redux/apiCalls.js

```diff
@@ -0,0 +1,32 @@
+// SHOW BOOKS
```

Comment:
- [ ] It will be nice to remove all commented code from your code-base.
- [ optional ]I can see your API key. It will be nice to hide it.

Reply:
This issue is well addressed. Nice work! :clap: